compose.yml

```diff
@@ -33,40 +54,32 @@ services:
     entrypoint: sh
     command: -c 'minio server /data --console-address ":9001" --address ":9002"'
 
-  app:
-    build:
-      context: .
-      dockerfile: ./docker/Dockerfile.dev
-    container_name: captable-app
-    restart: always
-    ports:
-      - 3000:3000
-    env_file:
-      - ".env.example"
-      - ".env"
-    volumes:
-      - .:/app
-      - /app/node_modules
-      - /app/.next
-    depends_on:
-      - pg
-      - smtp
-      - minio
-
   # minio client - is expected to be run once to setup minio buckets. It'll exit 0 after setup
   mc:
     image: minio/mc
     container_name: captable-minio-client
-    entrypoint: sh
-    command: -c "
-      mc alias set myminio http://minio:9002 captable password;
-      mc mb myminio/captable-private-bucket;
-      mc mb myminio/captable-public-bucket;
-      mc policy set public myminio/captable-public-bucket;
-      exit 0;
-     "
     depends_on:
       - minio
+    entrypoint: sh
+    command: |
+      -c "
+        sleep 10;
+        mc alias set myminio http://minio:9002 captable password;
+        mc mb myminio/captable-private-bucket;
+        mc mb myminio/captable-public-bucket;
+        mc policy set public myminio/captable-public-bucket;
+        exit 0;
+      "
+  adminer:
```

Comment:
This looks like a helpful tool to have. However with prisma ORM, we already have `prisma studio` which is the preferred one on next/node community. Can we get that going instead of `adminer`?

Here is the command to run it `pnpm db:studio` and it runs on port `5555`

Reply:
Oh, that is news to me. I was unaware of that capability. I will get rid of `adminer` then.